pyblish_lite/app.py

```diff
@@ -52,9 +55,23 @@ def show(parent=None):
 
         ctrl = control.Controller()
 
-        win = window.Window(ctrl, parent)
-        win.resize(430, 600)
-        win.show()
+        win = None
+        try:
+            win = self._window
+            win.show()
+            win.activateWindow()
```

Comment:
Possibly, yes.

The important bit is that you try the code out both when the window is (1) minimized and (2) not minimized but behind another window. If you get those right, ideally on both Qt 4 and 5, then we're home safe. Even it'll be for just Windows this time around (is it?). Accuracy in cosmetics (which this is) isn't as big of a deal than accuracy in functionality, so we can hop testing on Linux and OSX for the time being.